contracts/src/libraries/PlonkVerifier.sol

```diff
@@ -87,57 +87,32 @@ library PlonkVerifier {
     /// @param verifyingKey The Plonk verification key
     /// @param publicInput The public input fields
     /// @param proof The TurboPlonk proof
-    /// @param extraTranscriptInitMsg Optional bytes for transcript init message
     /// @return _ A boolean indicating successful verification, false otherwise
     function verify(
         IPlonkVerifier.VerifyingKey memory verifyingKey,
         uint256[] memory publicInput,
-        IPlonkVerifier.PlonkProof memory proof,
-        bytes memory extraTranscriptInitMsg
+        IPlonkVerifier.PlonkProof memory proof
     ) internal view returns (bool) {
         _validateProof(proof);
-        for (uint256 i = 0; i < publicInput.length; i++) {
-            BN254.validateScalarField(BN254.ScalarField.wrap(publicInput[i]));
-        }
-        PcsInfo[] memory pcsInfos = new PcsInfo[](1);
-        pcsInfos[0] = _preparePcsInfo(verifyingKey, publicInput, proof, extraTranscriptInitMsg);
-        return _batchVerifyOpeningProofs(pcsInfos);
-    }
 
-    /// @dev Batch verify multiple TurboPlonk proofs.
-    /// @param verifyingKeys An array of verifier keys
-    /// @param publicInputs A two-dimensional array of public inputs.
-    /// @param proofs An array of Plonk proofs
-    /// @param extraTranscriptInitMsgs An array of bytes from
-    /// transcript initialization messages
-    function batchVerify(
-        IPlonkVerifier.VerifyingKey[] memory verifyingKeys,
-        uint256[][] memory publicInputs,
-        IPlonkVerifier.PlonkProof[] memory proofs,
-        bytes[] memory extraTranscriptInitMsgs
-    ) internal view returns (bool) {
-        if (
-            verifyingKeys.length != proofs.length || publicInputs.length != proofs.length
-                || extraTranscriptInitMsgs.length != proofs.length || proofs.length == 0
-        ) {
-            revert InvalidPlonkArgs();
-        }
-
-        PcsInfo[] memory pcsInfos = new PcsInfo[](proofs.length);
-        for (uint256 i = 0; i < proofs.length; i++) {
-            // validate proofs are proper group/field elements
-            _validateProof(proofs[i]);
-            // validate public input are all proper scalar fields
-            for (uint256 j = 0; j < publicInputs[i].length; j++) {
-                BN254.validateScalarField(BN254.ScalarField.wrap(publicInputs[i][j]));
+        // This *if* statement is needed in order for some tests to pass.
```

Comment:
can you share more on this?

Reply:
when i run 
`forge test --match-contract PlonkVerifier_verify_Test --ffi -vvvvv`

I get
```
Encountered 1 failing test in contracts/test/PlonkVerifier.t.sol:PlonkVerifier_verify_Test
[FAIL. Reason: panic: assertion failed (0x01)] test_verify_succeeds() (gas: 531359)
```

Reply:
I don't get any error when running this command. Did you recompile the rust code before running those tests? 

```
cargo build --bin diff-test --release
```

Reply:
i'd advocate against this anti-pattern.

we can simply accept `function fuzzTest(publicInputs: uint256[8])` in the test instead of changing production contract to placate test logic.

Reply:
Agreed. Change done in 5d55f589.